app/views/profiles/_show.html.erb

```diff
@@ -70,6 +70,26 @@
       <p>Storage Space Description: <%= partner_profile.describe_storage_space %></p>
       <br>
     <% end %>
+
+    <% if partner_profile_fields.include?('area_served') || partner_profile_fields.empty? %>
+      <h2 class='text-2xl underline'>Area Served</h2>
+      <table>
+        <th class="county-heading">County</th><th class="county-heading">% of Clients in county</th>
+
+        <% served_area_col = Partners::ServedArea.where(partner_profile: partner_profile) %>
+        <% if served_area_col.any? %>
+          <%= render partial: "partners/profiles/show/served_area_row", collection: served_area_col %>
+        <% else %>
+          <tr ><td class="county-name"><%= "No County Specified" %></td><td class="client-share percent">100%</td></tr>
+        <%end %>
+      </table>
+      <br>
+    <% end %>
+
+
+
+
+
```

Comment:
Lots of extra space here :)

Reply:
Apparently I need the erb equivalent of rubocop.  Fixing for the next push.